src/main/java/bar/Simplest.java

```diff
@@ -5,6 +5,8 @@
  */
 public class Simplest {
 
+    public int FOO = 2;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make FOO a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://rebloch.sonarsource.com/organizations/fab/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://rebloch.sonarsource.com/project/issues?id=test%3Afake-project-for-tests&pullRequest=1&issues=AWHcBRsZQYL-lH1H_cI8&open=AWHcBRsZQYL-lH1H_cI8)

[ISSUE_KEY]: AWHcBRsZQYL-lH1H_cI8